src/Microsoft.Azure.SignalR.Protocols/ConnectionMessage.cs

```diff
@@ -171,6 +171,8 @@ public ConnectionDataMessage(string connectionId, ReadOnlySequence<byte> payload
         /// Gets or sets the payload is partial or not.
         /// </summary>
         public bool IsPartial { get; set; }
+
+        public byte PartitionKey { get; }
```

Comment:
why not `public byte PartitionKey => GeneratePartitionKey(ConnectionId);`

Reply:
To avoid duplicate calculation?

Reply:
use `=>` for less size